scripts/l10n.ts

```diff
@@ -16,6 +16,21 @@ const GIT_REPO = 'mui-x';
 const L10N_ISSUE_ID = 3211;
 const SOURCE_CODE_REPO = `https://github.com/${GIT_ORGANIZATION}/${GIT_REPO}`;
 
+const packagesWithL10n = [
+  {
+    key: 'data-grid',
+    reportName: '🧑‍💼 DataGrid, DataGridPro, DataGridPremium',
+    constantsRelativePath: 'packages/grid/x-data-grid/src/constants/localeTextConstants.ts',
+    localesRelativePath: 'packages/grid/x-data-grid/src/locales',
+  },
+  {
+    key: 'pickers',
+    reportName: '🕒 Date and Time Pickers',
```

Comment:
To match with DataGrid.

```suggestion
    reportName: '📅🕒 Date and Time Pickers',
```